packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactRootView.java

```diff
@@ -434,10 +434,8 @@ public void run() {
 
     if (mShouldLogContentAppeared) {
       mShouldLogContentAppeared = false;
-
-      if (mJSModuleName != null) {
```

Comment:
What's the motivation for this change? It looks safer to check if it's defined than to assert if it isn't. I'd rather miss this marker than make the app crash.